### PR TITLE
Add github action to check eslint and step identifier doubles for UI tests

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       |  1.7.7.x
| Description?  | Add github action to check eslint and step identifier doubles for UI tests (this jobs only runs if Label TE is set on PR)
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | Should a new github workflow in checks sub tab 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21122)

### CODE REVIEW

**Review by SimonGrn — DISMISSED** [review]

File: .github/workflows/ui-tests.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Install dependencies in UI tests directory
+        run: npm install
+
+      - name: Check eslint errors
+        run: npm run lint-no-fix
```
Comment (Progi1984) [inline .github/workflows/ui-tests.yml]:
```suggestion
        run: npm run lint-dry-run
```

File: tests/UI/package.json [hunk 2]
```diff
@@ -7,11 +7,13 @@
   },
   "scripts": {
     "linkchecker": "./node_modules/mocha/bin/mocha campaigns/tools/linkchecker.js",
-    "lint": "eslint  --fix --ignore-path .gitignore .",
+    "lint": "eslint --fix --ignore-path .gitignore .",
+    "lint-no-fix": "eslint --ignore-path .gitignore .",
```
Comment (Progi1984) [inline tests/UI/package.json]:
```suggestion
    "lint-dry-run": "eslint --ignore-path .gitignore .",
```

**Review by Progi1984 — DISMISSED** [review]

File: tests/UI/package.json (hunk 2 above)
Comment (NeOMakinG) [reply tests/UI/package.json]: On the core we use lint, and lint-fix, that would be good to collapse with others !

Comment (boubkerbribri) [reply tests/UI/package.json]: Done thanks

**Review by Progi1984 — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**